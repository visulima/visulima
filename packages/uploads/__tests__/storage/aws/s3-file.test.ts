import {
    describe, expect, it,
} from "vitest";

import S3File from "../../../src/storage/aws/s3-file";

describe("S3File", () => {
    it("should have extra params", () => {
        const file = new S3File({ metadata: { foo: "bar" } });

        file.uri = "uri";
        file.Parts = [];
        file.UploadId = "UploadId";
        file.partsUrls = [];
        file.partSize = 1;
        file.id = "8sUP8GVW46Ijxg2wPuhaK";
        file.originalName = "8sUP8GVW46Ijxg2wPuhaK",

        expect(file).toMatchSnapshot();
    });
});
