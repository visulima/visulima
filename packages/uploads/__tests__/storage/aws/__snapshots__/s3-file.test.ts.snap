// Vitest Snapshot v1

exports[`S3File > should have extra params 1`] = `
S3File {
  "Parts": [],
  "UploadId": "UploadId",
  "bytesWritten": NaN,
  "content": undefined,
  "contentType": "application/octet-stream",
  "createdAt": undefined,
  "expiredAt": undefined,
  "hash": undefined,
  "id": "8sUP8GVW46Ijxg2wPuhaK",
  "metadata": {
    "foo": "bar",
  },
  "modifiedAt": undefined,
  "name": "",
  "originalName": "8sUP8GVW46Ijxg2wPuhaK",
  "partSize": 1,
  "partsUrls": [],
  "size": undefined,
  "status": undefined,
  "uri": "uri",
}
`;
